secure-api-gateway-ob-uk-rs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rs/server/api/obie/account/v4_0_0/statements/StatementsApiController.java

```diff
@@ -159,8 +170,38 @@ public ResponseEntity<Object> getAccountsAccountIdStatementsStatementIdFile(Stri
     }
 
     @Override
-    public ResponseEntity<OBReadTransaction6> getAccountsAccountIdStatementsStatementIdTransactions(String statementId, String accountId, String authorization, String xFapiAuthDate, String xFapiCustomerIpAddress, String xFapiInteractionId, String xCustomerUserAgent, String apiClientId, String consentId, int page) {
-        return null;
+    public ResponseEntity<OBReadTransaction6> getAccountsAccountIdStatementsStatementIdTransactions(String statementId, String accountId, String authorization, String xFapiAuthDate, LocalDateTime fromBookingDateTime, LocalDateTime toBookingDateTime, String xFapiCustomerIpAddress, String xFapiInteractionId, String xCustomerUserAgent, String apiClientId, String consentId, int page)
+            throws OBErrorException {
+        logger.info("getAccountStatementTransactions from account id {}, statement id {}, consentId: {}, apiClientId: {} fromBookingDate {} toBookingDate {} pageNumber {} ",
```

Comment:
debug?

Reply:
to keep it consistent with the v3.1.10 I would keep it this way. Might help debugging into sandbox environments where cloud logs are set to error only.

Reply:
Hmmm... ok. That's going to make for very chatty logging though, and a lot of log storage, etc.

Why is the sandbox only error level? We should be able to manipulate that to debug if we need help debugging.

i.e. It's kind of the wrong way round.

Reply:
For now though, if it's helpful, go ahead. 

I think this should be addressed at some point though.